src/site/layouts/tests/vamc/health_care_region_page.unit.spec.js

```diff
@@ -1,16 +1,26 @@
 import { expect } from 'chai';
 import { parseFixture, renderHTML } from '~/site/tests/support';
+import axeCheck from '~/site/tests/support/axe';
 
 const layoutPath = 'src/site/layouts/health_care_region_page.drupal.liquid';
 
 describe('intro', () => {
   describe('no fieldTitleIcon', () => {
+    let container;
     const data = parseFixture(
       'src/site/layouts/tests/vamc/fixtures/health_care_region_page.json',
     );
 
-    it('renders elements with expected values', async () => {
-      const container = await renderHTML(layoutPath, data);
+    before(async () => {
+      container = await renderHTML(layoutPath, data);
+    });
+
+    it('reports no axe violations', async () => {
+      const violations = await axeCheck(container);
+      expect(violations.length).to.equal(0);
```

Comment:
Can we encapsulate these 2 lines in a single function?

Reply:
@mpelzsherman assuming your idea is for the `axeCheck` to probably to throw an error if there are violations? (And the error's message probably being the stringified list)